docker/zap-api-scan.py

```diff
@@ -518,7 +519,7 @@ def main(argv):
                     f.write('# Only the rule identifiers are used - the names are just for info\n')
                     f.write('# You can add your own messages to each rule by appending them after a tab on each line.\n')
                     for key, rule in sorted(all_dict.items()):
-                        f.write(key + '\tWARN\t(' + rule + ')\n')
+                        f.write(f'{key}\tWARN\t({rule})\n')
```

Comment:
python 3.6 will be out of support from 2021-12. If we want to retain it, we can't use f-strings. Have your say.